joern-cli/frontends/gosrc2cpg/src/main/scala/io/joern/gosrc2cpg/astcreation/AstForExpressionCreator.scala

```diff
@@ -54,8 +53,12 @@ trait AstForExpressionCreator(implicit withSchemaValidation: ValidationMode) { t
   }
 
   private def astForStarExpr(starExpr: ParserNodeInfo): Seq[Ast] = {
-    val cNode   = createCallNodeForOperator(starExpr, Operators.indirection)
     val operand = astForNode(starExpr.json(ParserKeys.X))
+    val typeFullName = operand.head.root.get.properties
```

Comment:
These `head.root.get` seem unsafe. See if something like this works:
```scala
operand.headOption
  .flatMap(_.root)
  .map(_.properties.get(PropertyNames.TYPE_FULL_NAME))
  .getOrElse(Defines.anyTypeName)
  .toString
```

Reply:
done